docs/topic-guides/hp-tuning-det/hp-adaptive-advanced.txt

```diff
@@ -22,15 +22,14 @@ Search mode:
 
 Resource budget:
 
--  ``target_trial_steps``: The maximum number of steps that any trial
-   that survives to the end of the experiment will be trained for (a
-   *step* is a `fixed number <batches-per-step>`__ of batches). This quantity
-   is domain-specific and should roughly reflect the number of training steps
-   needed for the model to converge on the data set. For users who would like
-   to determine this number experimentally, train a model with reasonable
-   hyperparameters using the ``single`` search method.
--  ``step_budget``: Set ``step_budget`` to roughly 10 times
-   ``target_trial_steps``. A higher ``step_budget`` will result in
+-  ``max_length``: The maximum training length of any trial that survives to
+   the end of the experiment. This quantity is domain-specific and should
+   roughly reflect the training length needed for the model to converge on
```

Comment:
"the training length needed" -> "the number of minibatches the model must be trained on for it to converge"